glean-core/csharp/Glean/Glean.cs

```diff
@@ -311,5 +319,57 @@ private void InitializeCoreMetrics()
             string storedvalue = GleanInternalMetrics.architecture.TestGetValue();
             Console.WriteLine("InitializeCoreMetrics - has value {0} and that's {1}", hasValue, storedvalue);
         }
-    }
+
+        /// <summary>
+        /// Register the pings generated from `ManualPings` with the Glean SDK.
+        /// </summary>
+        /// <param name="pings"> The `Pings` object generated for your library or application
+        /// by the Glean SDK.</param>
+        private void RegisterPings(object pings)
+        {
+            // Instantiating the Pings object to send this function is enough to
+            // call the constructor and have it registered through [Glean.registerPingType].
+            Console.WriteLine("Registering pings");
```

Comment:
Did you check if, when getting in the body of this function, the `GleanInternalPings` get lazily instantiated? (that's the whole point of this function)

Reply:
yes. I checked.